zebra-consensus/src/chain.rs

```diff
@@ -33,16 +33,23 @@ use zebra_chain::parameters::{Network, NetworkUpgrade::Sapling};
 
 /// The maximum expected gap between blocks.
 ///
-/// Used to identify unexpected high blocks.
+/// Used to identify unexpected out of order blocks.
 const MAX_EXPECTED_BLOCK_GAP: u32 = 100_000;
```

Comment:
Non-blocking on this PR:

IIRC this logic was added to help debug sync issues.  Do we still need it, if we sync correctly and can handle out of order blocks?  If not, could we simplify the chain verifier by removing it?

Reply:
I think we should remove the out of order debug logs after:
* the state service handles out of order blocks #1011
* the inbound service handles out of order gossiped blocks #889

I added this task to the checklist in the sync correctness tracking issue #884.